### PR TITLE
Change old UI link to React App to point to graph page

### PR DESCRIPTION
**Pull request by boyskila**
This PR should fix https://github.com/prometheus/prometheus/issues/6685

### CODE REVIEW

**Comment by boyskila** [comment]
@juliusv

File: web/ui/react-app/src/Navbar.tsx [hunk 1]
```diff
@@ -31,7 +31,17 @@ const Navigation: FC<PathPrefixProps> = ({ pathPrefix }) => {
             </NavLink>
           </NavItem>
           <NavItem>
-            <NavLink tag={Link} to={`${pathPrefix}/new/graph`}>
+            <NavLink
+              tag={Link}
+              to={`${pathPrefix}/new/graph`}
+              onClick={e => {
```
Comment (juliusv) [inline web/ui/react-app/src/Navbar.tsx]: Hmm TBH I've always used the "Graph" link at the top to get to a clean/new graph page without any panels on it, since that's the quickest way (otherwise you have to edit the URL to remove the query string, or navigate somewhere else and back to clear the page). So I'm kind of a fan of keeping that behavior (is there a good use case for the new behavior, since it effectively makes the button a no-op? What does the `navigate(...)` with the query string do btw. - since in that case we are not on the graph page, we also wouldn't have an existing query string, so why append it?

Comment (boyskila) [reply web/ui/react-app/src/Navbar.tsx]: Hm, but currently the query string is the only thing that disappears and I thought that this is a bug and this is what ```navigate``` does - prevents this by attaching the QS back. Somehow this can't be achieved by just doing ```to={`${pathPrefix}/new/graph${window.location.search}`}```

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "go.inferGopath": false
+}
```
Comment (juliusv) [inline .vscode/settings.json]: seems like this unrelated change made it into the PR

Comment (boyskila) [reply .vscode/settings.json]: yes, will remove it

Comment (boyskila) [reply .vscode/settings.json]: Done.

**Comment by juliusv** [comment]
:+1:
